nntrainer/utils/node_exporter.h

```diff
@@ -207,11 +207,21 @@ class Layer;
 /**
  * @copydoc template <typename PropsType, typename NodeType> void
  * Exporter::saveTflResult(const PropsType &props, const NodeType *self);
+ * @todo remove this when layer is refactored
```

Comment:
This will be still needed, even though it might be an empty function.

Reply:
as layer will become stateless, this won't be needed.

Reply:
The reason is that default specialization is throwing operation. Supported functions (or operations) are whitelisted in this fashion.

Reply:
done.